src/Shared/LayoutRenderers/AspNetRequestCookieLayoutRenderer.cs

```diff
@@ -37,6 +39,24 @@ public class AspNetRequestCookieLayoutRenderer : AspNetLayoutMultiValueRendererB
         /// </summary>
         public List<string> CookieNames { get; set; }
 
+        /// <summary>
+        /// Gets or sets the keys to exclude from the output. If omitted, none are excluded.
+        /// </summary>
+        /// <docgen category='Rendering Options' order='10' />
+#if ASP_NET_CORE
+        public ISet<string> Exclude { get; set; }
+#else
+        public HashSet<string> Exclude { get; set; }
+#endif
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AspNetRequestCookieLayoutRenderer" /> class.
+        /// </summary>
+        public AspNetRequestCookieLayoutRenderer()
+        {
+            Exclude = new HashSet<string>(new[] { "AUTH", "SESS_ID" }, StringComparer.OrdinalIgnoreCase);
```

Comment:
I like the StringComparer.OrdinalIgnoreCase, but I'm not sure if that works when exclude options are set from config :/

Reply:
That's probably an existing issue then.

Reply:
I guess changes could be make to fix this. But will make that a separate PR.

Reply:
👍

Reply:
-> https://github.com/NLog/NLog/issues/3591